drakrun/drakrun/main.py

```diff
@@ -288,12 +288,20 @@ def generate_graphs(self, workdir):
     def slice_logs(self, workdir):
         plugin_fd = {}
 
-        with open(os.path.join(workdir, 'drakmon.log'), 'r') as f:
-            for line in f:
+        with open(os.path.join(workdir, 'drakmon.log'), 'rb') as f:
+            while True:
+                line = f.readline()
+
+                if not line:
+                    break
+
                 try:
-                    obj = json.loads(line)
+                    line_s = line.strip().decode('utf-8')
```

Comment:
Shouldn't the order be `line.decode('utf-8').strip()` ?

Reply:
`strip()` works on bytes as well:

```
>>> b"wtf\nlol\naaa".strip()
b'wtf\nlol\naaa'
```